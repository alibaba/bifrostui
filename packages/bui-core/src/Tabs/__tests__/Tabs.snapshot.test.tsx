import { snapshotTest } from 'testing';

describe('Tabs snapshot', () => {
  snapshotTest('Tabs');
});
