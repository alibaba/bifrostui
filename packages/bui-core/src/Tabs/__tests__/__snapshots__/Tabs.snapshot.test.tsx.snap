// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs snapshot Tabs demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-tabs"
    style={
      Object {
        "marginBottom": "12px",
      }
    }
  >
    <div
      className="bui-tabs-mask bui-tabs-mask-left"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-mask bui-tabs-mask-right"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-tabs"
      onScroll={[Function]}
    >
      <div
        className="bui-tabs-tabline"
        style={
          Object {
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      />
      <div
        className="bui-tab bui-tab-center bui-tab-active"
        onClick={[Function]}
      >
        水果
      </div>
      <div
        className="bui-tab bui-tab-center"
        onClick={[Function]}
      >
        蔬菜
      </div>
      <div
        className="bui-tab bui-tab-center"
        onClick={[Function]}
      >
        动物
      </div>
    </div>
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-active"
  >
    菠萝
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
</div>
`;

exports[`Tabs snapshot Tabs demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-tabs"
    style={
      Object {
        "marginBottom": "12px",
      }
    }
  >
    <div
      className="bui-tabs-mask bui-tabs-mask-left"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-mask bui-tabs-mask-right"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-tabs"
      onScroll={[Function]}
    >
      <div
        className="bui-tabs-tabline"
        style={
          Object {
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      />
      <div
        className="bui-tab bui-tab-center bui-tab-active"
        onClick={[Function]}
      >
        水果
      </div>
      <div
        className="bui-tab bui-tab-center bui-tab-disabled"
        onClick={[Function]}
      >
        蔬菜
      </div>
      <div
        className="bui-tab bui-tab-center"
        onClick={[Function]}
      >
        动物
      </div>
    </div>
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-active"
  >
    菠萝
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
</div>
`;

exports[`Tabs snapshot Tabs demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-tabs"
    style={
      Object {
        "marginBottom": "12px",
      }
    }
  >
    <div
      className="bui-tabs-mask bui-tabs-mask-left"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-mask bui-tabs-mask-right"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-tabs"
      onScroll={[Function]}
    >
      <div
        className="bui-tabs-tabline"
        style={
          Object {
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      />
      <div
        className="bui-tab bui-tab-center bui-tab-active bui-tab-disabled"
        onClick={[Function]}
      >
        水果
      </div>
      <div
        className="bui-tab bui-tab-center bui-tab-disabled"
        onClick={[Function]}
      >
        蔬菜
      </div>
      <div
        className="bui-tab bui-tab-center bui-tab-disabled"
        onClick={[Function]}
      >
        动物
      </div>
    </div>
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-active"
  >
    菠萝
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
</div>
`;

exports[`Tabs snapshot Tabs demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-tabs"
    style={
      Object {
        "marginBottom": "12px",
      }
    }
  >
    <div
      className="bui-tabs-mask bui-tabs-mask-left"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-mask bui-tabs-mask-right"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
    <div
      className="bui-tabs-tabs"
      onScroll={[Function]}
    >
      <div
        className="bui-tabs-tabline"
        style={
          Object {
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      />
      <div
        className="bui-tab bui-tab-active"
        onClick={[Function]}
        title="水果"
      >
        水果
      </div>
      <div
        className="bui-tab"
        onClick={[Function]}
        title="蔬菜"
      >
        蔬菜
      </div>
      <div
        className="bui-tab"
        onClick={[Function]}
        title="动物"
      >
        动物
      </div>
    </div>
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-active"
  >
    <div>
      菠萝
    </div>
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      onClick={[Function]}
      type="button"
    >
      我要蚂蚁
    </button>
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      onClick={[Function]}
      type="button"
    >
      我要西红柿
    </button>
  </div>
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
  <div
    className="bui-tabpanel bui-tabpanel-inactive"
  />
</div>
`;

exports[`Tabs snapshot Tabs demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    style={
      Object {
        "width": "325px",
      }
    }
  >
    <div
      className="bui-tabs"
      style={
        Object {
          "marginBottom": "12px",
          "marginTop": "20px",
        }
      }
    >
      <div
        className="bui-tabs-mask bui-tabs-mask-left"
        style={
          Object {
            "opacity": 0,
          }
        }
      />
      <div
        className="bui-tabs-mask bui-tabs-mask-right"
        style={
          Object {
            "opacity": 0,
          }
        }
      />
      <div
        className="bui-tabs-tabs"
        onScroll={[Function]}
      >
        <div
          className="bui-tabs-tabline"
          style={
            Object {
              "transform": "translate3d(0px, 0px, 0px)",
              "transition": "",
            }
          }
        />
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="长津湖"
        >
          长津湖
        </div>
        <div
          className="bui-tab bui-tab-center bui-tab-active"
          onClick={[Function]}
          title="战狼2"
        >
          战狼2
        </div>
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="你好，李焕英"
        >
          你好，李焕英
        </div>
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="哪吒之魔童降世"
        >
          哪吒之魔童降世
        </div>
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="流浪地球"
        >
          流浪地球
        </div>
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="唐人街探案3"
        >
          唐人街探案3
        </div>
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="复仇者联盟4：终局之战"
        >
          复仇者联盟4：终局之战
        </div>
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="长津湖之水门桥"
        >
          长津湖之水门桥
        </div>
        <div
          className="bui-tab bui-tab-center"
          onClick={[Function]}
          title="红海行动"
        >
          红海行动
        </div>
      </div>
    </div>
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
    <div
      className="bui-tabpanel bui-tabpanel-active"
    >
      2
    </div>
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
    <div
      className="bui-tabpanel bui-tabpanel-inactive"
    />
  </div>
</div>
`;
